src/tests/tokens/nf-token.test.ts

```diff
@@ -397,4 +397,14 @@ spec.test('throws when trying to burn non existant NFT', async (ctx) => {
   const id1 = ctx.get('id1');
 
   await ctx.reverts(() => nftoken.instance.methods.burn(id1).send({ from: owner }), '003002');
+});
+
+spec.test('safeTransfer does not call onERC721Received to constructing contract', async (ctx) => {
+  const sendsToSelfOnConstruct = await ctx.deploy({ 
+    src: './build/sends-to-self-on-construct.json',
+    contract: 'SendsToSelfOnConstruct',
+  });
+
+  const logs = sendsToSelfOnConstruct.logs;
+  ctx.is(logs.events.Received, undefined);
```

Comment:
Deploy is a bit different then other transactions. It returns an object with an `instance` which is an instance of the deployed contract and `receipt` with information about the deployment. Receipt is what you should look at. If receipt does not contain what you need try updating to the latest specron version: 0.16.0